app/src/main/res/values/strings.xml

```diff
@@ -201,6 +201,7 @@
   <string name="admin_auth_admin_controls_sub">Enter the Administrator PIN in order to access Administrator Controls.</string>
   <string name="admin_auth_pin">Administrator\'s PIN</string>
   <string name="admin_auth_incorrect">Incorrect Administrator PIN. Please try again.</string>
+  <string name="admin_auth_null">This field can\'t be empty.</string>
```

Comment:
@rt4914 Do we need to confirm this message with Chantel or does it sounds good?

Reply:
@mschanteltc If you check the video in the issue: https://github.com/oppia/oppia-android/issues/3134 you will see that if the learner clicks on `Submit` button if there is no text then there is no error message. Unfortunately we cannot `disable` the `Submit` button here so we will have to show error message only. 
So does `This field can't be empty` makes sense to me?
To me `Please enter Administrator Pin` makes more sense meaning something precise as an error.
WDYT?

Reply:
I agree that the latter choice `Please enter Administrator PIN` works better.